content/post/74-lisbon-hack-week.md

```diff
@@ -0,0 +1,30 @@
+---
+date: 2019-03-09
+url: 74-lisbon-hack-week
+tags: hack week ipfs libp2p community researchers
+title: 2018 Hack Week - Lisbon
+author: André Cruz
+---
+
+In the week of May 21, 2018, we hosted another Hack Week in the sunny city of Lisbon.
+
+The week was great for core developers, contributors and researchers to spend quality time together and work in a fast paced synchronous format towards solving concrete problems. It was incredible to see ideas and solutions flourishing for open problems we've been thinking for a while. The hacker spirit was strong throughout the entire week.
+
+The majority of discussions and collaborations were around IPFS, libp2p and applications built on top of them, but we also hosted a [CRDT Research meetup](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTVkko1ZTn-qcGb-n6EqHff) on the second day.
+
+Didn't have a chance to be present? No worries, we got your back. Many of the report back presentations given at the end of the week were record so that you can learn about what got developed, designed and decided. We have compiled them into a [YouTube playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ). You can see them all here:
```

Comment:
```suggestion
Didn't have a chance to be present? No worries, we got your back. Many of the report back presentations were recorded so that you can learn about what got developed, designed and decided. We have compiled them into a [YouTube playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtR1hDEAKYXtUguNBkg2qceQ). You can see them all here:
```